scripts/build/babel-plugins/transform-custom-require.js

```diff
@@ -22,7 +22,7 @@ module.exports = function(babel) {
         if (isEvalRequire(node.callee) && node.arguments.length === 1) {
           let arg = node.arguments[0];
           if (t.isLiteral(arg) && arg.value.startsWith(".")) {
-            const value = "." + arg.value.substring(arg.value.lastIndexOf("/"));
+            const value = "." + arg.value.slice(arg.value.lastIndexOf("/"));
```

Comment:
This one is not equal, since 'foo'.slice(-1) === 'o', 'foo'.substring(-1) === 'foo'

But `require` startsWith `.` always has `/`(except `require('.')` or `require('..')`), I think.